ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -15,7 +15,7 @@
 public class AdministrationPage extends Page {
 
     private static final String URL = "/admin/index.htm";
-    private static final By MANAGE_USERS = By.cssSelector("#legacyui-manageUsers a");
+    private static final By MANAGE_USERS = By.cssSelector("#content a[href='/openmrs/admin/users/users.list']");
```

Comment:
The current set property would be one of the best option among the alternatives. However, the element `attribute property value` is only true when inspected via qa instance, and inspecting it via `demo.openmrs.org` & also on the` sdk local instance` the set element `property value` is false. In those two instances have mentioned the element `value property` is set to `#legacyui-mangeUsers a`. For code sustainability and quality assurance, I opted to set the property value to the suggested one since it hardly breaks with any server instance either.